bcipy/helpers/load.py

```diff
@@ -167,7 +166,17 @@ def load_experimental_data() -> str:
     return filename
 
 
-def load_signal_model(filename: str = None):
+def load_signal_model(model_class: SignalModel, model_kwargs: Dict[str, Any], filename: str = None):
+    """Construct the specified model and load pretrained parameters.
+
+    Args:
+        model_class (SignalModel, optional): Model class to construct.
+        model_kwargs (dict, optional): Keyword arguments for constructing model.
+        filename (str, optional): Location of pretrained model parameters.
+
+    Returns:
+        SignalModel: Model after loading pretrained parameters.
```

Comment:
It returns a tuple (SignalModel, str). Let's add a return type to the method signature too.

```
def load_signal_model(model_class: SignalModel, model_kwargs: Dict[str, Any], filename: str = None) -> Tuple[SignalModel, str]:
```